Makefile

```diff
@@ -39,17 +39,13 @@ endif
 LD_FLAGS = $(BUILTIN_LD_FLAGS) $(EXTRA_LD_FLAGS)
 
 # BUILTIN_GC_FLAGS are the internal flags used to pass compiler.
-BUILTIN_GC_FLAGS ?= all=-trimpath="$$HOME"
+BUILTIN_GC_FLAGS ?=
 # EXTRA_GC_FLAGS are the caller-provided flags to pass to the compiler.
 EXTRA_GC_FLAGS =
 # GC_FLAGS are the union of the above two BUILTIN_GC_FLAGS and EXTRA_GC_FLAGS.
 GC_FLAGS = $(BUILTIN_GC_FLAGS) $(EXTRA_GC_FLAGS)
 
-ASM_FLAGS ?= all=-trimpath="$$HOME"
-
-# TRIMPATH contains arguments to be passed to go to strip paths on Go 1.13 and
-# newer.
-TRIMPATH ?= $(shell [ "$$($(GO) version | awk '{print $$3}' | sed -e 's/^[^.]*\.//;s/\..*$$//;')" -ge 13 ] && echo -trimpath)
+ASM_FLAGS ?=
```

Comment:
Could this be removed completely, since it seems like there's no need for it anymore?  It looks like it was introduced in 65b65d074de753f0166ce13cd6dba0695dcdd154 of #3443 just to support this use case, so maybe it could be dropped.